airflow/providers/cncf/kubernetes/operators/pod.py

```diff
@@ -238,6 +238,7 @@ class KubernetesPodOperator(BaseOperator):
         state, or the execution is interrupted. If True (default), delete the
         pod; if False, leave the pod.
         Deprecated - use `on_finish_action` instead.
+    :param termination_message_policy: set container termination_message_policy. Defaults to "File"
```

Comment:
Can you cover this behaviour using a unit test?

Reply:
I added a test